doc/radosgw/upgrade_to_jewel_with_different_region_pool.rst

```diff
@@ -0,0 +1,16 @@
+=============
+RGW upgrading to Jewel with a different rgw region root pool
+=============
+
+.. versionadded:: Jewel
+
+Upgrade of :term:`Ceph Object Gateway` to jewel with a different
+rgw region root pool. This document describes the needed configuration changes.
```

Comment:
might want to be more specific about when this applies. we should also mention the settings for realm/period pools. something like this?

```markdown
If an existing multisite configuration uses a non-default setting for
`rgw region root pool`, the new pool settings `rgw zonegroup root pool`,
`rgw period root pool`, and `rgw realm root pool` should be set to match.
```